src/items/mod.rs

```diff
@@ -19,6 +19,7 @@ mod test {
     use crate::{CheckMenuItem, IconMenuItem, MenuId, MenuItem, PredefinedMenuItem, Submenu};
 
     #[test]
+    #[cfg_attr(all(miri, not(target_os = "linux")), ignore)]
```

Comment:
why does these examples need to be ignored on Windows and macOS using miri?

Reply:
Because on MacOS and windows these tests involve FFI which is not supported by Miri at this time.

MacOS: https://github.com/pcwizz/muda/actions/runs/6826919902/job/18567898544#step:7:181
Windows: https://github.com/pcwizz/muda/actions/runs/6826919902/job/18567898285#step:7:124

Reply:
I see, thanks